Automattic/jetpack
# Simple Payments: Only fire init hooks if enabled

There exists a `jetpack_disable_simple_payments` filter, but it does not prevent the init hooks from firing, thus the CPT and shortcode are still registered.

#### Changes proposed in this Pull Request:

* Check if Simple Payments are enabled before use.

This is likely incomplete. It looks like right now we still want the shortcode enabled so we can display an warning to an admin when it is disabled. May want to juggle things around a bit.

#### Jetpack product discussion


None at this time.

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Add `add_filter( 'jetpack_disable_simple_payments', '__return_true' );` to a connected Jetpack site with a higher plan.
* Check the list of CPTs on the site to see if `jp_pay_order` is included or not.
